app/services/session_encryptor.rb

```diff
@@ -1,4 +1,6 @@
 class SessionEncryptor
+  MARSHAL_SIGNATURE ||= 'BAh'.freeze
```

Comment:
More importantly, see https://github.com/18F/identity-idp/pull/1358/commits/6e66dec36c6a6986cedd976e8b0953aa4f61f6c5 for what this constant means.